halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ListCommand.java

```diff
@@ -0,0 +1,29 @@
+package com.netflix.spinnaker.halyard.cli.command.v1.config;
+
+import com.netflix.spinnaker.halyard.cli.command.v1.config.AbstractConfigCommand;
+import com.netflix.spinnaker.halyard.cli.services.v1.Daemon;
+import com.netflix.spinnaker.halyard.cli.services.v1.OperationHandler;
+import com.netflix.spinnaker.halyard.cli.ui.v1.AnsiFormatUtils;
+import com.netflix.spinnaker.halyard.config.model.v1.node.DeploymentConfiguration;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.Getter;
+
+public class ListCommand extends AbstractConfigCommand {
+    @Getter(AccessLevel.PUBLIC)
+    private String commandName = "list";
+
+    @Getter(AccessLevel.PUBLIC)
+    private String description = "Lists all deployments";
+
+    @Override
+    protected void executeThis() {
+        new OperationHandler<List<DeploymentConfiguration>>()
+            .setFailureMesssage("Failed to get all deployments.")
+            .setSuccessMessage("Retrieved all deployments.")
+            .setFormat(AnsiFormatUtils.Format.JSON)
```

Comment:
The default for other commands has been either YAML or a special string -- any reason to choose JSON?

Reply:
Yeah, YAML was giving me trouble displaying the deployments. Was getting this error:
```
! ERROR Cannot deserialize instance of `java.util.LinkedHashMap`
  out of START_ARRAY token
 at [Source: UNKNOWN; line: -1, column: -1]
```

Reply:
Ohh we might be assuming that the object being deseralized is always a map, and in this case it's a list. Could you add a check in the deserializer to handle that correctly?